commit 47d94a5db982dc9bc3f4ae213d263fcca480b2c4 (refs/changes/69/469/4)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-09-15T14:48:41-07:00 (3 years, 1 month ago)
    
    MORAY-358 use a livelier url to reference node-fast
    
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/README.md b/README.md
index a5981fb..94531de 100644
--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 This repo contains Moray, the highly-available key/value store from Joyent.
 Moray offers a simple put/get/del (as well as search) protocol on top of
-Postgres 9.x, over plain TCP see <https://github.com/mcavage/node-fast>.
+Postgres 9.x, over plain TCP see <https://github.com/joyent/node-fast>.
 
 This repository is part of the Joyent SmartDataCenter project (SDC), and the
 Joyent Manta project.  For contribution guidelines, issues, and general
